src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -562,13 +593,23 @@ def BuildDpa(dpa_name, protection_points_method=None):
     IOError: if the provided file cannot be found.
     ValueError: if the provided file is not a valid GeoJSON MultiPoint geometry.
   """
```

Comment:
Are you going to generate the geojson files containing the protected points for each DPA?

Reply:
Not really (as it is the role of ITS/NTIA/NIST to do so) ??
Of course from the generate points with my simple method, it is easy to create a json file with:
```
with open(filename, 'w') as fd:
   json.dump(sgeo.MultiPoint([pt for pt in dpa.protected_points]))
```